src/agent/src/rpc.rs

```diff
@@ -1438,7 +1438,7 @@ fn find_process<'a>(
     Ok(p)
 }
 
-pub fn start<S: Into<String>>(s: Arc<Mutex<Sandbox>>, host: S, port: u16) -> ttrpc::Server {
+pub fn start(s: Arc<Mutex<Sandbox>>, server_address: String) -> ttrpc::Server {
```

Comment:
Here the server_address parameter can be used as '&str' type,  since it was used as read only.

Reply:
Good point @lifupan - fixed!